hnn_core/network.py

```diff
@@ -473,7 +473,8 @@ def __eq__(self, other):
                                 'clear_connectivity', 'clear_drives',
                                 'connectivity', 'copy', 'gid_to_type',
                                 'plot_cells', 'set_cell_positions',
-                                'to_dict', 'write_configuration'])
+                                'to_dict', 'write_configuration',
+                                'update_weights'])
```

Comment:
We need to add new Network methods to this list so that it doesn't try to check it like an attribute.